src/App.jsx

```diff
@@ -53,7 +60,9 @@ export default function App() {
             <CoinPage {...props}/>
```

Comment:
looks into link and fetching data via url params like in the weather app

Reply:
fixed with useParams hook